### PR TITLE
grpc for PodList

### PR DESCRIPTION
**Pull request by Helen-Xie**
signed-off-by: Xie Yulin xylin821@163.com

### CODE REVIEW

**Comment by Helen-Xie** [comment]
I submmited a new Pull request, so I closed the previous Pr #134

**Comment by resouer** [comment]
@Helen-Xie This implementation is something wrong. If test it against latest hypernetes and hyperd, it will report "Can not get Pod info with pod name()".

You should add id to get pod info: `req := grpctypes.PodInfoRequest{PodID: pod.PodID}`

File: pkg/kubelet/hyper/hyperclient.go [hunk 1]
```diff
@@ -141,6 +141,14 @@ type podRemoveResult struct {
 	ID    string `json:"ID"`
 }
 
+type HyperPod struct {
```
Comment (resouer) [inline pkg/kubelet/hyper/hyperclient.go]: Why not use HyperPod directly just like other functions did? `hyper/types.go` and this belongs to the same package ...

**Comment by resouer** [comment]
lgtm

**Pull request merged**